### PR TITLE
chore: Hide App Components from Nav Drawer

### LINKED ISSUE #1113: Hide App Components
### Chore

**Description**

The EventBrite does not offer the following options which is present in Orga app's Drawer Menu  -
1. Faq
2. Sponsor
3. Tracks
4. Feedback
5. Speakers
6. SpeakersCall
7. About 

According to Mario, these functionalities make our app more complicated for new users who migrate from EventBrite to Orga App. Thus, these should be removed from Dashboard but the code need not be deleted because they can be incorporated in future.

**Would you like to work on the issue?**

Yes

**CC:** @iamareebjamal

### PR DESCRIPTION
**Pull request by Masquerade0097**
Fixes #1113 

Checklist:

- [x] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Changes: 
1. Hide items mentioned in Issue tracker from Navigation Drawer.

Screenshots for the change:
drawer

### CODE REVIEW

**Comment by srv-twry** [comment]
@Masquerade0097 Please update your branch though.

**Comment by iamareebjamal** [comment]
Please do so that items are still visible in Debug mode to help developers

**Comment by Masquerade0097** [comment]
@iamareebjamal I have commented out code for later use.

**Comment by iamareebjamal** [comment]
Simply create a list of entries for debug version and if not DEBUG mode, hide those items

**Comment by Masquerade0097** [comment]
@iamareebjamal Ok, Added build config variable to show drawer items only in debug mode.

**Comment by sridharjajoo** [comment]
@Masquerade0097 Could you also change the position of Edit event to below the Orders as it was discussed in the meeting

**Comment by Masquerade0097** [comment]
@sridharjajoo Yes, did it.

File: app/src/main/java/org/fossasia/openevent/app/core/main/MainActivity.java [hunk 1]
```diff
@@ -181,4 +186,14 @@ private void loadFragment(int navItemId) {
...
     }
+
+    private void hideDrawerItems() {
+        binding.navView.getMenu().findItem(R.id.nav_feedback).setVisible(false);
+        binding.navView.getMenu().findItem(R.id.nav_faq).setVisible(false);
+        binding.navView.getMenu().findItem(R.id.nav_track).setVisible(false);
+        binding.navView.getMenu().findItem(R.id.nav_sponsor).setVisible(false);
+        binding.navView.getMenu().findItem(R.id.nav_speaker).setVisible(false);
+        binding.navView.getMenu().findItem(R.id.nav_speakers_call).setVisible(false);
+        binding.navView.getMenu().findItem(R.id.nav_about_event).setVisible(false);
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/core/main/MainActivity.java]: Simply create a list of ids above and iterate through them to hide

File: app/src/main/java/org/fossasia/openevent/app/core/main/MainActivity.java [hunk 2]
```diff
@@ -36,6 +41,8 @@
 
     public static final String EVENT_KEY = "event";
     private long eventId = -1;
+    private List<Integer> drawerItems = new ArrayList<>(Arrays.asList(R.id.nav_feedback, R.id.nav_faq, R.id.nav_track,
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/core/main/MainActivity.java]: No need to wrap in new ArrayList. Arrays.asList is enough

File: app/src/main/java/org/fossasia/openevent/app/core/main/MainActivity.java [hunk 3]
```diff
@@ -133,6 +140,12 @@ public void setEventId(long eventId) {
         this.eventId = eventId;
         fragmentNavigator.setEventId(eventId);
         binding.navView.getMenu().setGroupVisible(R.id.subMenu, true);
+
+        if (BuildConfig.HIDE_DRAWER_ITEMS) {
+            for (Integer i : drawerItems) {
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/core/main/MainActivity.java]: Rename to menuId

**Pull request merged**